python/dgl/graph_index.py

```diff
@@ -650,7 +681,6 @@ def adjacency_matrix(self, transpose, ctx):
         else:
             raise Exception("unknown format")
 
-    @utils.cached_member(cache='_cache', prefix='inc')
```

Comment:
the same here.

Reply:
Same as adjmat, we ask users to cache them.